mmdet/models/backbones/hrnet.py

```diff
@@ -210,12 +211,52 @@ class HRNet(nn.Module):
             memory while slowing down the training speed.
         zero_init_residual (bool): whether to use zero init for last norm layer
             in resblocks to let them behave as identity.
+
+    Example:
+        >>> from mmdet.models.backbones.hrnet import *  # NOQA
```

Comment:
May also adopt the same option.

Reply:
I rebased on the latest master and fixed this issue.